Std/Lean/Meta/Basic.lean

```diff
@@ -390,7 +390,7 @@ then runs `f` again on all of those goals, and repeats until `f` fails on all re
 or until `maxIters` total calls to `f` have occurred.
 Fails if `f` does not succeed at least once.
 -/
-def repeat1' [Monad m] [MonadError m] [MonadMCtx m]
+def repeat1' [Monad m] [MonadError m] [MonadExcept ε m] [MonadBacktrack s m] [MonadMCtx m]
```

Comment:
I noticed that `repeat1'` is never actually used.  Instead `Std.Tactic.Basic` defines a tactic `repeat1` (with no apostrophe) that uses `repeat1'` directly.

Should `repeat1'` here be removed?

Should the tactic `repeat1` be renamed to `repeat1'` for consistency since both apply to all subgoals?

Reply:
This is a public API function (I think it is used by mathlib as well), so I would rather not remove it. On the contrary I think it would be better to just call it from `repeat1`, so I've done that now.

Reply:
> Should the tactic repeat1 be renamed to repeat1' for consistency since both apply to all subgoals?

I've done this, although this is a breaking change, I'm sure `repeat1` gets a fair amount of usage. Maybe we should have an alias for it or something?

Reply:
Mathlib seems to compile fine against this branch as is.